src/client/components/pages/AddMeal/AddMeal.js

```diff
@@ -0,0 +1,128 @@
+import React, { useState } from "react";
+import "./AddMeal.css";
+
+
+const initialValues = {
+    title: "",
+    location: "",
+    description: "",
+    maxReservations: "",
+    when: "",
+    price: "",
+};
+
+
+const AddMeal = () => {
+    const [inputValues, setInputValues] = useState(initialValues);
+
+
+    const handleOnChange = (e) => {
+        const { name, value } = e.target;
+        setInputValues({ ...inputValues, [name]: value });
+    };
+
+    async function onSubmit(e) {
+        e.preventDefault();
+        const meal = {
+            title: inputValues.title,
+            description: inputValues.description,
+            location: inputValues.location,
+            when: inputValues.when,
+            max_reservations: inputValues.maxReservations,
+            price: inputValues.price,
+            created_date: date,
+        };
+
+
+        try {
+            await fetch ("/api/meals", meal);
+            const messagge = `Great, ${meal.title} is added`;
+            alert(messagge);
+        } catch {
+            alert("Your meal is not added...");
+            throw new Error("Your meal is not added, try again");
+        }
+        //making inputs empty
+        setInputValues(initialValues);
+    }
```

Comment:
You can use object destructuring to make this function simpler:

```suggestion
    async function onSubmit(e) {
        e.preventDefault();
        const {title, description, location, when, maxReservations, price} = inputValues;
        
        const meal = {
            title,
            description,
            location,
            when,
            max_reservations: maxReservations,
            price,
            created_date: date,
        };


        try {
            await fetch ("/api/meals", meal);
            const messagge = `Great, ${title} is added`;
            alert(messagge);
        } catch {
            alert("Your meal is not added...");
            throw new Error("Your meal is not added, try again");
        }
        //making inputs empty
        setInputValues(initialValues);
    }
```